common/config/rush/pnpm-lock.yaml

```diff
@@ -58,7 +58,7 @@ importers:
       '@itwin/appui-abstract': 4.0.0_@itwin+core-bentley@4.0.0
       '@itwin/appui-layout-react': 4.1.0_1c726c265f9cde6743ef19e93af5aeaa
       '@itwin/appui-react': 4.1.0_c4fc7958281cb1014a865ba80498a059
-      '@itwin/breakdown-trees-react': file:../../packages/itwin/breakdown-trees_29ec31b409c64649c3a02bf8d7d1122e
+      '@itwin/breakdown-trees-react': file:../../packages/itwin/breakdown-trees_2a5c7b3246bd2f72b770f8e93a49e389
```

Comment:
im surprised how little this file changed. can you run `rush update` again and commit the modified lockfile if any?

Reply:
I did a couple times to be sure. I can do it again.

Reply:
No additional changes to the lock file after `rush update`.